packages/vue/src/components/authenticator.vue

```diff
@@ -125,6 +126,48 @@
       </template>
     </confirm-sign-up>
 
+    <reset-password
+      v-if="actorState?.matches('resetPassword')"
+      @reset-password-submit="onResetPasswordSubmitI"
+      ref="resetPasswordComponent"
+    >
+      <template #resetPasswordSlotI>
+        <slot name="reset-password"></slot>
+      </template>
+      <template
+        #footer="{ info, onResetPasswordSubmit, onBackToSignInClicked }"
+      >
+        <slot
```

Comment:
Thanks for staying ahead and creating all these slots! I've been pretty sloppy with slots on Angular, and I'll have to catch up quickly.

Reply:
👍 I've been trying to keep up slots for all the big components. I suspect we'll need more as the customer gets a hold of it.